src/convene.html

```diff
@@ -1,8 +1,8 @@
 ---
 layout: layouts/product-landing-page.njk
 title: Convene
-subtitle: Bringing Workers Together
-summaryText: Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui lorem cupidatat commodo. Elit sunt amet fugiat veniam occaecat fugiat aliqua.
+subtitle: &mdash; Video-based Online Collaboration Tools
```

Comment:
Is `&mdash` a unicode for a dash? I am just curious on why isn't it showing on the screenshot.

Reply:
Ah, I think it's getting stripped out. It's a unicode symbol for an [em-dash](https://www.compart.com/en/unicode/U+2014)